server/utils/mailer.js

```diff
@@ -0,0 +1,15 @@
+const aws = require('../system/aws')
+
+const i18n = require('../system/i18n')
+
+const sendEmail = async (to, msgKey, msgParams = {}, lang) => {
+  i18n.translate(async t => {
```

Comment:
why can't you simply use ```i18nFactory.createI18nPromise``` ? do we really need a new file?

Reply:
Using i18nFactory